src/server/room-controller.js

```diff
@@ -46,6 +46,10 @@ export default class RoomController {
     const { user } = req;
     const { token } = req.params;
 
+    if (!user) {
```

Comment:
This can never be the case (unless passport has a bug) as this handler is protected by our `needs-permission-middleware.js` which in turn calls `req.isAuthenticated()` which in turn already does the same check: https://github.com/jaredhanson/passport/blob/master/lib/http/request.js#L69

Reply:
Please see comment here 👉 https://github.com/educandu/educandu/pull/189#discussion_r793389400